### ISSUE TITLE
Supported Reminder For Email Attachments

### ISSUE THREAD

**Issue by 4jNsY6fCVqZv** [elementary/mail#425]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Feature
**Is your feature request related to a problem? Please describe.**
Unfortunately it happens far too often and it would be nice if Mail would support me in sending a mail only once.

**Describe the solution you'd like**
Mail should ask on the basis of keywords whether I would like to add the appropriate attachment if I should forget it. It would be wonderful if I could edit the corresponding keywords myself, if necessary.

**Existing work**
I am aware that Evolution mail offers a plugin for this.

**Comment by jlnr** [comment]
I've mentioned this in #232, but having a separate issue is definitely better :+1:
